### DISCUSSION TITLE
Welcome to NoiseTorch Discussions!

### DISCUSSION THREAD

**Discussion by ZyanKLee** [noisetorch/NoiseTorch#263] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Foexle11** [comment]
Hi, I'm Foexle, I'm quite frankly an ordinary user, who hates introduction-rounds ^^". Right now I'm just amazed how the Noisetorch projekt is somehow evolving and waiting for my time to somehow help, without being a "rocket-scientist" (from my perspective, cause I nearly understand a third of whats going on XD )
So yea, thats me. 
You can now flood the commends, that the first move is made :P

**Reply by ZyanKLee** [reply]
Welcome @Foexle11, thanks for the kind words. There will surely be a time, when non-developers like you and me can be of help in this project. ;-)
